.github/scripts/setup-env.sh

```diff
@@ -41,9 +41,10 @@ conda create \
   python="${PYTHON_VERSION}" pip \
   ninja cmake \
   libpng \
+  openssl=3.0.9 \
   'ffmpeg<4.3'
 conda activate ci
-conda install --quiet --yes libjpeg-turbo -c pytorch
+conda install --quiet --yes libjpeg-turbo openssl=3.0.9 -c pytorch
```

Comment:
Why do we need it again here? Is this just there to avoid updating it with the new installation? If so, why does `libjpeg-turbo` pull in `openssl` in the first place?